pom.xml

```diff
@@ -35,8 +35,14 @@ See the Apache License Version 2.0 for the specific language governing permissio
 		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
 		<enforcer.version>3.1.0</enforcer.version>
 		<java.enforced.version>[11,12)</java.enforced.version>
+		<!-- source and target used by tycho plugin -->
 		<maven.compiler.source>11</maven.compiler.source>
 		<maven.compiler.target>11</maven.compiler.target>
```

Comment:
Might as well get rid of `source` and `target`, as it is pretty confusing that they're here while not being used.